sdk/resourcemanager/azure-resourcemanager-appservice/src/test/java/com/azure/resourcemanager/appservice/FunctionAppsTests.java

```diff
@@ -76,6 +77,7 @@ protected void cleanUpResources() {
     }
 
     @Test
+    @DoNotRecord(skipInPlayback = true)
```

Comment:
why?

Reply:
The test cases `canCRUDFunctionApp` and `canCRUDLinuxFunctionApp` can work in `RECORD` and `LIVE` modes, but not working in `PLAYBACK` mode (the local machine too). The detail infomation following as URL:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=3267132&view=results

Reply:
Not a valid reason. CI works before with the Playback. Please check with Xiaofei on the recording.

Reply:
Fixed in the new version.